src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPage.tsx

```diff
@@ -1,21 +1,81 @@
 import AccordionSection from '@common/components/AccordionSection';
 import Details from '@common/components/Details';
 import RelatedInformation from '@common/components/RelatedInformation';
-import themeService, { Theme } from '@common/services/themeService';
+import themeService, {
+  PublicationSummary,
+  Theme,
+} from '@common/services/themeService';
 import Link from '@frontend/components/Link';
 import Page from '@frontend/components/Page';
 import PageSearchFormWithAnalytics from '@frontend/components/PageSearchFormWithAnalytics';
 import { GetServerSideProps, NextPage } from 'next';
-import React from 'react';
+import React, { Fragment } from 'react';
 import { logEvent } from '@frontend/services/googleAnalyticsService';
 import Accordion from '@common/components/Accordion';
+import { ReleaseType, releaseTypes } from '@common/services/types/releaseType';
 import PublicationList from './components/PublicationList';
 
 interface Props {
   themes: Theme[];
 }
 
 const FindStatisticsPage: NextPage<Props> = ({ themes = [] }) => {
+  const {
+    NationalStatistics,
+    OfficialStatistics,
+    ...releaseTypesExcludingNationalAndOfficalStatistics
+  } = releaseTypes;
+
+  const publicationTypes = {
+    ...releaseTypesExcludingNationalAndOfficalStatistics,
+    NationalAndOfficial: 'National and official statistics',
+    Legacy: 'Not yet on this service',
+  } as const;
+
+  type PublicationType = keyof typeof publicationTypes;
+
+  const releaseTypeToPublicationType = (
+    releaseType?: ReleaseType,
+  ): PublicationType => {
+    switch (releaseType) {
+      case 'NationalStatistics':
+      case 'OfficialStatistics':
+        return 'NationalAndOfficial';
+      case undefined:
```

Comment:
Pushed mapping to back-end after creating a new `PublicationType` enum which simplifies things. Publication tree view model now has a `Type` field rather than `LatestReleaseType`.